fp/2022-2023/fp-p1/proj_tester.py

```diff
@@ -164,7 +308,7 @@ def equal(x,y):
 def get_lastest_commit_hash():
     try:
         result = requests.get(
-            "https://api.github.com/repos/diogotcorreia/proj-ist-unit-tests/commits?path=fp%2F2022-2023%2Ffp-p1%2Fproj_tester.py&page=1&per_page=1"
+            "https://api.github.com/repos/SrGesus/proj-ist-unit-tests/commits?path=fp%2F2022-2023%2Ffp-p1%2Fproj_tester.py&page=1&per_page=1"
```

Comment:
I don't think you should change this for this repository

Reply:
Eu tinha mudado esqueci de fazer push